### PR TITLE
Implement some suggestions

### PR DESCRIPTION
**Pull request by PeterJFB**
- Add color to links
- Add image field to articles
- Create css globals (move these to next ui later)
- Make navbar consistent
- Organize articles based on before/after arrival

### CODE REVIEW

**Review by ollfkaih — APPROVED** [review]
LGTM

File: components/splashPage.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { Button } from '@nextui-org/react';
-import Link from 'next/link';
+import { Button, Pagination, Text } from '@nextui-org/react';
 import styles from '../styles/Home.module.css';
```
Comment (norbye) [inline components/splashPage.tsx:3]: Prefer consistency in where the style files are located. Currently this file is located in /styles, but all the other components you add have styles in /components

File: styles/Home.module.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .topContainer {
-  min-height: 100vh;
+  height: 100%;
```
Comment (norbye) [inline styles/Home.module.css:2]: Is there a reason you changed this? Afaik the new value has no impact, so it could rather be deleted.

**Review by norbye — APPROVED** [review]
Looks nice!

Do you have any thoughts on the filename conventions?  
Personally I like when components with multiple files have their own folder, and for component file/folder naming to use PascalCase to match the components name.

File: styles/Home.module.css (hunk 2 above)
Comment (PeterJFB) [reply styles/Home.module.css]: A (flex)-element's height would by default only be as high as its content. Making it 100% ensures children are able to use the available space, which makes sense for the `Home`-component :) 100vh also solves this issue, but the way it was implemented here led to some unnecessary scrolling:P

File: components/splashPage.tsx (hunk 1 above)
Comment (PeterJFB) [reply components/splashPage.tsx]: :100: agree, I was just a bit lazy:P `.modules` is now the norm :)

**Pull request merged**